crates/ethrpc/src/types.rs

```diff
@@ -240,6 +240,9 @@ pub enum SignedTransaction {
     /// Signed ERC-1559 transaction.
     #[serde(rename = "0x2")]
     Erc1559(SignedErc1559Transaction),
+    /// Signed EIP-4844 transaction.
+    #[serde(rename = "0x3")]
+    Eip4844(SignedEip4844Transaction),
```

Comment:
Regarding your comment in the PR description - it looks like this was added to the `execution-apis` repo [5 days ago](https://github.com/ethereum/execution-apis/commit/057892b256c181232ca22499d1801b9637192698) - it is part of the RPC changes that will come with the next hardfork (Dencun).

Personally, I'm OK with leaving it in here since it will need to be added at some point, and I don't think it affects the current API.